src/Compiler/TypedTree/TypedTreePickle.fs

```diff
@@ -1499,6 +1499,7 @@ let u_MemberFlags st : SynMemberFlags=
       IsDispatchSlot=x4
       IsOverrideOrExplicitImpl=x5
       IsFinal=x6
+      IsCompilerGenerated=false
```

Comment:
A thing to note here is that we're not pickling this new field and according to Don this is ok. 

Adding it to the serialization would change the format and probably have long reaching consequences.